.github/PROCESS.md

```diff
@@ -85,6 +86,57 @@ We have two long-living branches:
 - `master`: completed features, bug fixes, refactors, chores
 - `stable`: the latest release
 
+The overall flow:
```

Comment:
It seems a bit odd to have the flow summary start with "creating a release". Also this seems more of like an overview of the document than a flow?